ci/genezio.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+#
+# This workflow will install genezio, then run genezio deploy for your project.
+#
+# To use this workflow, you will need to complete the following set-up steps:
+#
+# 1. Create a free genezio account at https://genez.io
+#
+# 2. Store a genezio access token in GitHub Actions secrets named `GENEZIO_TOKEN`.
+#
+# See the documentation for more details and best practices.
+# https://docs.genez.io/genezio-github-action
+
+name: Deploy project with genezio
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+      - uses: actions/checkout@v3
+      - name: Setup Node.js
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install genezio
+      - uses: Genez-io/genezio-github-action@9bc4335eee1341ce0b03df58cd459a54f631ea6a # v1.0.1
+        with:
+          genezio-version: latest
+          token: ${{ secrets.GENEZIO_TOKEN }}
+      - name: Deploy with genezio
+        working-directory: ./server
+        run: genezio deploy
```

Comment:
here I would put `genezio deploy --backend` and then another step with `genezio deploy --frontend`

Reply:
Done. :D